src/_sass/rouge.scss

```diff
@@ -204,6 +204,94 @@
 .highlight .w {
   color: #bbbbbb;
 }
-.highlight {
-  background-color: #f8f8f8;
-}
+// .highlight {
+//   background-color: #f8f8f8;
+// }
```

Comment:
commented this out to keep consistent with the dark mode & play nicely with default Primer styling